tests/integrations/mcs/scheduling/server_test.go

```diff
@@ -675,3 +675,70 @@ func (suite *multipleServerTestSuite) TestReElectLeader() {
 	rc = suite.pdLeader.GetServer().GetRaftCluster()
 	rc.IsPrepared()
 }
+
+func (suite *serverTestSuite) TestOnlineProgress() {
```

Comment:
nit , please move TestOnlineProgress into `serverTestSuite` scope instead of behind multipleServerTestSuite :)